scripts/globals/shop.lua

```diff
@@ -508,7 +507,6 @@ xi.shop =
                 4376,     120,      xi.ki.RHAPSODY_IN_WHITE,   -- Meat Jerky
                 4371,     184,      xi.ki.RHAPSODY_IN_WHITE,   -- Grilled Hare
                 4381,     720,      xi.ki.RHAPSODY_IN_UMBER,   -- Meat Mithkabob
-                4456,     550,      xi.ki.RHAPSODY_IN_WHITE,   -- Boiled Crab
```

Comment:
Suggest commenting out instead of deleting: SE is likely to put these back after they _learn to do math_ and change some values.